libs/SmartSync/SmartSync/Classes/Util/SFSyncUpTarget.m

```diff
@@ -128,7 +128,7 @@ - (void)fetchRecordModificationDates:(NSDictionary *)record
     };
     
     id completeBlock = ^(NSDictionary *response) {
-        if (nil != response) {
```

Comment:
Blindly accessing array index 0 would cause the crash when the remote record was gone